src/search/screens/search.screen.js

```diff
@@ -294,11 +306,20 @@ class Search extends Component {
       this.getSearchResults().length === 0 &&
       !this.getSearchPagination().isFetching;
 
+    let isLocation = '';
+
+    if (params) {
+      if (params.location) {
+        isLocation = `location:${params.location}`;
```

Comment:
`isLocation` usually means a boolean. How about naming it as `searchText`? And we can check in only one if statement, `if (params && params.location)`.